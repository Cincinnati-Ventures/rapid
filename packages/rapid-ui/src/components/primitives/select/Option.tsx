import React from 'react';

interface OptionProps extends React.OptionHTMLAttributes<HTMLOptionElement> {}

const Option = (props: OptionProps) => {
	return <option {...props} />;
};

export default Option;
