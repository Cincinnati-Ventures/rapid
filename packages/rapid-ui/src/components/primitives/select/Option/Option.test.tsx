import React from 'react';
import { render } from '@testing-library/react';
import { expect, it, describe } from 'vitest';
import Option from './Option';

describe('Option', () => {
	it('should render correctly', () => {
		const { container } = render(<Option />);
		expect(container).toMatchSnapshot();
	});
});
