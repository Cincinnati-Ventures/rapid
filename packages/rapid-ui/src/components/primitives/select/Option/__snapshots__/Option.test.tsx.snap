// Vitest Snapshot v1

exports[`Option > should render correctly 1`] = `
<div>
  <option />
</div>
`;
